flows/contact.go

```diff
@@ -130,6 +140,18 @@ func (c *Contact) SetLanguage(lang envs.Language) { c.language = lang }
 // Language gets the language for this contact
 func (c *Contact) Language() envs.Language { return c.language }
 
+// IsBlocked returns whether the contact is blocked or not
+func (c *Contact) IsBlocked() bool { return c.isBlocked }
```

Comment:
`Blocked` / `SetBlocked` / `Stopped` / `SetStopped` (go convention does prefix getters with `get` or `is` etc)